apis/sgv2-docsapi/src/main/resources/application.yaml

```diff
@@ -71,6 +71,8 @@ quarkus:
       bridge:
         host: localhost
         port: 8091
+        retry: true
```

Comment:
Are the default settings (max retries, delay etc) good here? We have quite low latency between API and Bridge. Although I guess actual time operations take varies a lot (may have big queries) so may be difficult to select low timeout.

Reply:
I did not want to touch the dafults for now, although there are 5 retires set. That's actually the only setting.